lnbits/core/crud.py

```diff
@@ -42,63 +41,23 @@
 # --------
 
 
-async def create_user(
```

Comment:
Note: `create_user` and `create_account` were overlapping. There is now a function in `services` called `create_user_account` which encapsulates the logic from here